### PR TITLE
Add machine learning model code

### PR DESCRIPTION
**Pull request by peiranli**
This pull request contains everything of the machine learning model training code. Currently, there is only code and the docstrings will be added soon.

### CODE REVIEW

File: model/load_data.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    def __init__(self):
+        self.pr_level_features = None
+        self.file_level_features = {}
+        self.reverted = False
+
+
+class DataLoader:
+    def __init__(self, repos):
+        self.repos = repos
+        self.pr_columns = PR_LEVEL_FEATURES + ['num new files', 'num files',
```
Comment (shail229) [inline model/load_data.py]: Why you can't add these extra features to PR_LEVEL_FEATURES constant? Or use two constants:
COMMON_ PR_LEVEL_FEATURES
& 
PR_LEVEL_FEATURES = COMMON_PR_LEVEL_FEATURES + extra features.

File: model/load_data.py [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+                                               'num reverted files',
+                                               'has reverted files',
+                                               'reverted files percentage']
+        self.file_columns = FILE_LEVEL_FEATURES + \
+                            ['reverted pull request percentage']
```
Comment (shail229) [inline model/load_data.py]: Same here, either add this extra feature to the constant.

**Review by shail229 — APPROVED** [review]
Thanks for adding these code to the repository. Please add the method doc for public methods and classes if you get time.

File: model/load_data.py [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+
+
+class CLData:
+    def __init__(self):
+        self.pr_level_features = None
+        self.file_level_features = {}
+        self.reverted = False
+
+
+class DataLoader:
```
Comment (jinhongcgoogle) [inline model/load_data.py:43]: Add doc to this class? This seems to be used across multiple files/modules.

File: model/load_data.py (hunk 2 above)
Comment (peiranli) [reply model/load_data.py]: Fixed.

File: model/load_data.py (hunk 1 above)
Comment (peiranli) [reply model/load_data.py]: Fixed.

File: model/load_data.py (hunk 3 above)
Comment (peiranli) [reply model/load_data.py:43]: Fixed.

**Pull request merged**